tools/release-note.md

```diff
@@ -15,6 +15,8 @@ use-case best.
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/grafana/agent/${RELEASE_TAG}/production/kubernetes/install.sh)" | kubectl apply -f -
 ```
 
+(Requires `envsubst` (GNU gettext).)
```

Comment:
Can this be moved before the script and not made a parenthetical? Something like: 

> The following script will download a Kubernetes manifest for the Agent and prompt for `remote_write` credentials. It requires `curl` and `envsubst` (GNU `gettext`).